app/controllers/indicator_questions_controller.rb

```diff
@@ -0,0 +1,57 @@
+class IndicatorQuestionsController < ApplicationController
+
+  # Callback Methods
+  before_action :set_indicator_question, only: [:show, :edit, :update, :destroy]
+
+
+  def index
+  end
+
+  def show
+  end
+
+  def new
+    @indicator_question = IndicatorQuestion.new
+  end
+
+  def create
+    @indicator_question = IndicatorQuestion.new(indicator_question_params)
+    if @indicator_question.save
+      # QUESTION: Should we truncate the indicator text so it only shows the first 5 words?
+      flash[:notice] = "Successfully added '#{@indicator_question.indicator.description}' indicator to this question."
+      @question = @indicator_question.question
+      # This assumes we add/delete IndicatorQuestions in the show question page
+      redirect_to question_path(@question)
+    else
+      # QUESTION: How should we handle redirection after failed saves for many to many?
```

Comment:
this should probably also go to the question_path(@question) path, since that's where you would have added the indicator_question. 
But make sure to add to the flash[:error] for this case (flash[:error] was not included in other controllers, because the form pages show from the model errors, not flash error, but all other pages just show the flash error)